pennylane/qnode.py

```diff
@@ -31,6 +31,21 @@
 from pennylane.tape import JacobianTape, QubitParamShiftTape, CVParamShiftTape, ReversibleTape
 
 
+def _jax_float64_support():
+    """Checks if float64 is enabled for jax.  Returns True of float64 support enabled,
+    False if not.
+    """
+    import jax.numpy as jnp
+
+    with warnings.catch_warnings():
+        warnings.filterwarnings("error")
+        try:
+            jnp.array(0.1, dtype=jnp.float64)
+        except UserWarning:
+            return False
+    return True

```

Comment:
I don't know how to test this line, given float64 mode must be configured on startup.